### ISSUE TITLE
Add new `boolean` parameter for getting liquidity provisions

### ISSUE THREAD

**Issue by AnExsomnis** [vegaprotocol/frontend-monorepo#3239]
Liquidity provisions tables were changed in the core, as well a `boolean` indicator to the Get method was added.
The PR and the specific change is here - https://github.com/vegaprotocol/vega/pull/7832/files#diff-7f6c5ea53fa01231a8a36ffa313993450073ba666c91f11fe0322f33a39b1b86R2115-R2120

And here is a follow up PR containing the GQL schema changes - https://github.com/vegaprotocol/vega/pull/7909

The `Get` method now uses a another parameter to indicate if the listed liquidity provision orders are the current live ones or all existing.

**Comment by JonRay15** [comment]
If this change is to a shared component of Console / LP Dashboard maybe we (LP Dashboard) will get this for free once this work is done. Would be good to confirm.  If not I've created another ticket for the LP Dashboard specific work.

**Comment by mattrussell36** [comment]
@JonRay15 We have already updated our queries on Console to use the live flag. However, I don't think the LP dashboard will get those changes due to splitting out the project to get it embedded on the website. 

@ciaran- can you confirm?

I'm removing the 'Trading' tag for now so its off our board
